pages/services/edge-lb/1.4/cloud-connector/index.md

```diff
@@ -0,0 +1,652 @@
+        "type": "NLB",
+        "internal": false,
+        "listeners": [
+          {
+            "port": 80,
+            "linkFrontend": "echo"
+          }
+        ],
+        "tags": [
+          {
+            "key": "Protocol",
+            "value": "HTTP"
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+As illustrated in this example, the `cloudProvider` field includes a subfield that identifies a specific cloud provider. In this case, the cloud provider subfield is `aws` to specify integration with an AWS Network Load Balancer. 
+
+The `aws` cloud provider field also has a subfield of `elbs`. The `elbs` field contains the configuration settings for a particular load balancer. The settings in this example define an AWS NLB configuration using the following fields and values:
+
+- `name` - Specifies the name that Edge-LB uses to generate names of the load balancers and their resources.
+
+- `type` - Specifies a load balancer type. Currently, only NLB is supported.
+
+- `internal` - Indicates whether the corresponding load balancer is for internal requests (`true`) or not (`false`). 
+
+  If the `internal` setting is `true`, the load balancer routes requests from internal clients running within the same cluster. 
+
+  If a load balancer is internet-facing with the `internal` field set to `false`, the load balancer can route external requests that are received from clients over the internet.
+  
+- `listeners` - Defines the following configuration details for each listener that receives inbound requests for the Edge-LB pool:
+  - `port` specifies a port number on which the respective load balancer is to listen for incoming connections from clients.
+
+  - `linkFrontend` specifies the name of the individual HAProxy frontend of the pool to which inbound requests are routed.
+
+- `tags` - Specifies an array of user-defined tag name and value pairs. The tags you specify using this field are applied to load balancers and target groups in addition to the [internal tags](#internal-tagging) that are automatically defined by the Edge-LB API server.
+
+## Specifying subnets
+You can define the subnets that are associated an AWS Network Load Balancer either manually or automatically.
+
+To specify subnets manually, you can add a `subnets` field to the cloud provider configuration details in the Edge-LB pool configuration file. For example, the following code snippet illustrates how to manually specify `subnet-1234567890abcdefgi` for the cloud provider load balancer:
+
+```json
+{
+  "name": "echo",
+  "type": "NLB",
+  "subnets": [
+    "subnet-1234567890abcdefgi"
+  ],
+  "listeners": [
+    {
+      "port": 80,
+      "linkFrontend": "echo"
+    }
+  ]
+}
+```
+
+If you don't specify the subnets you want attached to the AWS Network Load Balancer manually, then subnets are discovered automatically based on the **metadata** provided by Edge-LB pool instances. For more information about Edge-LB pool metadata, see [metadata]().
```

Comment:
metadata is not linked to content like https://github.com/mesosphere/dcos-edge-lb/blob/master/docs/METADATA.md?